### PR TITLE
Rewrite file URLs to force CloudFront > S3 bypassing our server

### PR DESCRIPTION
**Pull request by kduffie**

### CODE REVIEW

File: src/card-manager.ts [hunk 1]
```diff
@@ -1447,7 +1447,11 @@ export class CardManager implements Initializable, NotificationHandler, CardHand
         }
         const userProperties = await db.findCardProperties(card.id, "user", user.id);
         for (const property of userProperties) {
-          card.state.user.properties[property.name] = property.value;
+          let value = property.value;
+          if (typeof value === 'string') {
+            value = fileManager.rewriteFileUrls(value as string);
+          }
+          card.state.user.properties[property.name] = value;
         }
```
Comment (pshihn) [inline src/card-manager.ts]: ALL user string properties cannot be URLs (in the future)

Comment (pshihn) [reply src/card-manager.ts]: I see you are returning the same string when not URL. Never mind then. Just hard to read. 
Maybe in the long term do this automatically on DB insertion?

Comment (pshihn) [reply src/card-manager.ts]: Rest looks fine

**Comment by kduffie** [comment]
Sorry this isn't completely readable.  This is just a temporary step to
rewrite URLs until we complete the move to fileIDs.  See my emails from
this weekend.

**Pull request merged**